docs/FAQ/FAQ-General.md

```diff
@@ -139,3 +140,8 @@ A BTCPay Server functionality that is not found in the Greenfield API documentat
 ## How can I backup my BTCPay Server?
 
 It's possible to [create backups of your BTCPay Server](https://github.com/btcpayserver/btcpayserver-docker#how-can-i-back-up-my-btcpay-server) instance and it's data. Please keep in mind that the backup scripts have not been thoroughly tested for all types of BTCPay Server setups and custom deployments. Be sure to use your backup to verify it properly recreates your setup, before relying on it.
+
+## How can I charge for using my BTCPay Server instance?
+
+Currently charging someone for using your BTCPay Server instance, whether it's a percentage of transactions or a registration fee, is not supported natively.
```

Comment:
```suggestion
Currently charging users for using your BTCPay Server instance, whether it's a percentage of transactions or a registration fee, is not supported natively.
```

Reply:
An alternative way is to charge an upfront fee for a private link to register on the server. This does not require any technical skills. Link to https://docs.btcpayserver.org/FAQ/FAQ-ServerSettings/#how-to-add-a-new-user-by-invite

Reply:
> An alternative way is to charge an upfront fee for a private link to register on the server.

Yeah but that's not a BTCPay feature.

Otherwise committing your change.